src/app/pages/charging-stations/charging-limit/charging-profile-limit/charging-station-charging-profile-limit-chart.component.ts

```diff
@@ -130,6 +144,9 @@ export class ChargingStationSmartChargingLimitPlannerChartComponent implements O
           y: chargingSlot.limitInkW,
         } as number & ChartPoint);
       }
+      if (this.selectedUnit === ConsumptionUnit.AMPERE) {
+        chargingSlotDataSet.data.forEach((chartPoint) => chartPoint.y = chartPoint.y * 1000 / 230 ); // Utils convert method was updated. Needs to be handled by Serge.
```

Comment:
It's not that hard to just merge with the master-qa branch and follow the API changes of the method: it's just one argument added to the conversion function.